knowledge-base/textbox-add-icon.md

```diff
@@ -27,7 +27,13 @@ I would like to add a icon in the TextBox, like a search icon, or email or phone
 
 Is there anyway to do an 'addon' button with the TelerikTextBox?
 
-## Solution
+## Solution 1
+
+As of **UI for Blazor 5.1.0**, you may use [Prefix and Suffix templates]({%slug common-features/input-adornments%}) to add the desired content as prefix or suffix to the TextBox. 
+
+## Solution 2
+
+This solution applies to **UI for Blazor 5.0.x or earlier**.
```

Comment:
I guess the solution applies to all versions, but we recommend the other one?

Reply:
Shall I add a note in the first solution stating that it is the recommended one?